tasks/install.yml

```diff
@@ -11,7 +11,7 @@
     name: "{{ alertmanager_user }}"
     group: "{{ alertmanager_group }}"
     system: yes
-    shell: /sbin/nologin
+    shell: /usr/sbin/nologin
```

Comment:
It will be better to add this as a variable (Ex: `alertmanager_user_shell`) and declare it at [defaults/main.yml)](https://github.com/idealista/prometheus_alertmanager_role/blob/master/defaults/main.yml#L15), in this way we can change the value in each playbook independently.

Reply:
I just uploaded a change with this configuration.

Reply:
Great!